.github/workflows/api.yml

```
@@ -0,0 +1,34 @@
+name: API
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./api
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - uses: actions/setup-node@v2
+      with:
+        node-version: '16.2.0'
+
```

You can introduce cache here: 
```suggestion
      with:
        node-version: '16.2.0'
        cache: 'npm'
        cache-dependency-path: package-lock.json
```

---

Nice!